frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -164,6 +164,7 @@ main {
     }
 
     .usa-nav__primary button {
+        height: 100%;
```

Comment:
What is this for? I'm worried that if a button is in a large container the button will expand and be HUGE.

Reply:
This is for the "Admin" dropdown button within the nav. All of the items in the nav had a height of 100% except for this one.